app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<h1>Dashboard</h1>
+<table border="1">
+  <thead>
+    <th>Tenant</th>
+    <th>API Key</th>
+    <th>API Request Count</th>
+  </thead>
+  <tbody>
+    <% @tenants.each do |tenant|%>
+      <tr>
+        <td><%= tenant.name%></td>
+        <td><%= tenant.api_key %></td>
+        <td><%= tenant.tenant_api_requests.map(&:count).sum %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+<br/>
+<table border="1">
+  <thead>
+    <th>User</th>
+    <th>Questions</th>
+    <th>Answers</th>
+  </thead>
+  <tbody>
+    <% @users.each do |user| %>
+      <tr>
+        <td>
+          <%= user.name %>
+        </td>
+        <td>
+          <table>
+            <thead></thead>
+            <tbody>
+              <% user.questions.each_with_index do |question, index| %>
+                <tr>
+                  <td>
+                    <%= index + 1 %>. <%= question.title %> <b><%= question.private ? '(Private)' : ''%></b>
+                  </td>
+                </tr>
+              <% end %>
+            </tbody>
+          </table>
+        </td>
+        <td>
+          <table>
+            <thead></thead>
+            <tbody>
+              <% user.answers.each_with_index do |answer, index| %>
+                <tr>
+                  <td>
+                    <%= index + 1 %>. <%= answer.body %>
+                  </td>
+                </tr>
+              <% end %>
+            </tbody>
+          </table>
+        </td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
```

Comment:
Screen shot:
I think it hits the requirements as they are a little vague.  Displays all this data, in tables - not sure if aggregates where required

![screen shot 2017-08-17 at 12 17 10 pm](https://user-images.githubusercontent.com/2149063/29429598-1d1cb69e-8346-11e7-9803-6a24ff5b0712.png)